aio/scripts/start-cluster.sh

```diff
@@ -67,24 +61,17 @@ function start-ci-heapster {
   say "\nHeapster is up and running"
 }
 
-function start-local-minikube {
-  say "\nStarting local cluster"
-  sudo -E ${MINIKUBE_BIN} start --kubernetes-version=${MINIKUBE_K8S_VERSION}
-}
-
-function start-minikube {
+function start-kind {
+  ${KIND_BIN} create cluster --name="k8s-cluster-ci"
+  sleep 5
```

Comment:
Why `sleep` is required?

Reply:
It's not, I was testing something else and accidentally left it in. Thanks for the find! Removed.